src/plugins/auto/src/plugin.cpp

```diff
@@ -336,33 +290,13 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
         }
         return capabilities;
     }
-    auto val = m_plugin_config.get_property(name);
-    if (!is_new_api()) {
-        if (name == ov::hint::model_priority.name()) { // need to convert priority values to old API
-            ov::Any legacy_val{nullptr};
-            if (!val.empty()) {
-            switch (val.as<ov::hint::Priority>()) {
-                OPENVINO_SUPPRESS_DEPRECATED_START
-                case ov::hint::Priority::LOW: legacy_val = InferenceEngine::PluginConfigParams::MODEL_PRIORITY_LOW; break;
-                case ov::hint::Priority::MEDIUM: legacy_val = InferenceEngine::PluginConfigParams::MODEL_PRIORITY_MED; break;
-                case ov::hint::Priority::HIGH: legacy_val = InferenceEngine::PluginConfigParams::MODEL_PRIORITY_HIGH; break;
-                OPENVINO_SUPPRESS_DEPRECATED_END
-            default: OPENVINO_ASSERT(false, "Unsupported model priority value");
-            }
-        }
-        return legacy_val;
-        } else {
-            return val;
-        }
-    } else {
-        return val;
-    }
-    return val;
+    return m_plugin_config.get_property(name);
 }
 
 void Plugin::set_property(const ov::AnyMap& properties) {
     // with setConfig, only multi/auto supported internal configs can be accepted
-    m_plugin_config.set_property(pre_process_config(properties));
+    auto property_to_set = properties;
```

Comment:
@yangwang201911 if you still need time on this, maybe please create a seperate ticket? we can merge this PR first